schemas/CryptoSessionState.json

```diff
@@ -0,0 +1,26 @@
+{
+  "type": "object",
+  "description": "Describe the current session crypto state, that is if the session is verified or not, if recovery is correctly setup.",
+  "properties": {
+    "eventName": {
+      "enum": ["CryptoSessionState"]
+    },
+    "verificationState": {
+      "type": "string",
+      "oneOf": [
+        {"const": "Verified", "description": "The device is considered to be verified, it has been signed by its user identity."},
+        {"const": "NotVerified", "description": "The device is unverified."}
+      ]
+    },
+    "recoveryState": {
+      "type": "string",
+      "oneOf": [
+        {"const": "Enabled", "description": "Secret storage is set up and we have all the secrets locally."},
+        {"const": "Disabled", "description": "No default secret storage key exists or it is disabled explicitly using the account data event."},
+        {"const": "Incomplete", "description": "Secret storage is set up but we're missing some secrets."}
+      ]
+    }
+  },
+  "required": ["eventName","verificationState", "recoveryState"],
+  "additionalProperties": false
+}
```

Comment:
So IIUC, this Event will be used for state change, so maybe rename to something like `CryptoSessionStateChange` (not sure what's best)?